model/src/main/kotlin/config/LicenseFilenamePatterns.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.model.config
+
+import org.ossreviewtoolkit.utils.log
+
+data class LicenseFilenamePatterns(
+    /**
+     * A list of globs that match default license file names.
+     */
+    val licenseFilenames: List<String>,
+
+    /**
+     * A list of globs that match default patent file names. The patterns are supposed to be used case-insensitively.
+     */
+    val patentFilenames: List<String>,
+
+    /**
+     * A list of globs that match files that often define the root license of a project, but are no license files and
+     * are therefore not contained in [licenseFilenames]. The patterns are supposed to be used case-insensitively.
+     */
+    val rootLicenseFilenames: List<String>
+) {
+    /**
+     * A list of globs that match all kind of license file names, equaling the union of [licenseFilenames],
+     * [patentFilenames] and [rootLicenseFilenames]. The patterns are supposed to be used case-insensitively.
+     */
+    val allLicenseFilenames = (licenseFilenames + patentFilenames + rootLicenseFilenames).distinct()
+
+    companion object {
+        val DEFAULT = LicenseFilenamePatterns(
```

Comment:
These values should be put into `default.conf` instead. That's the purpose of the file, to define the default configuration which can be overridden by the user. So far it's empty, but this would be a good chance to start using it. I'm fine with doing it in a separate PR, but it should definitely be done.

Reply:
Would you keep this `DEFAULT` variable and obtain the values from that conf ?

Reply:
No, I would remove the variable and make `licenseFilePatterns` in `OrtConfiguration` non-nullable, because if it is not set by the user it is read from default.conf:
https://github.com/oss-review-toolkit/ort/blob/e14794e32274491fd9addaca95f611b058f66d2a/model/src/main/kotlin/config/OrtConfiguration.kt#L67

Reply:
Did you think about the usage from the tests? These are currently the main users of the default values.

Reply:
If you don't want to read the configuration file in the tests I think it you could easily create an instance of `LicenseFilenamePatterns` in the test code without relying on this `DEFAULT`.